<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>621</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
border-style: outset;
border-width: 2px;
border-color:rgb(27,25,39);
background-color:rgba(0, 192, 225, 85)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>901</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../img/Sfondo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>1281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:rgb(20, 18, 29)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="logoImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/Icona Information.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>790</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/Icona Close.png</normaloff>../img/Icona Close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
border-radius:15px;
background-color: rgb(51, 49, 77);
color:rgb(255, 255, 255);
}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
    </property>
    <property name="text">
     <string>Conferma</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/Icon confirm.png</normaloff>../img/Icon confirm.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>801</width>
     <height>821</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
border-style: outset;
border-width: 2px;
border-color:rgb(27,25,39);
background-color:rgba(0, 192, 225, 85)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Lettore</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fiscalCodeField">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Recapito</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contactField">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/Icona User.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/book.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Titolo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleField">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>ISBN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="isbnField">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Codice Fiscale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userField">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Autore</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="authorField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Tipologia Movimento</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="noteField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(23, 28, 78, 100);
color:rgb(255, 255, 255);
border-radius: 20px;
border-style: solid;
border:none;
text-align: center;
border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/Icona Movimenti.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeIni">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: black;
background-color:rgb(45, 206, 229);
border-radius:16px;</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEnd">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: black;
background-color:rgb(45, 206, 229);
border-radius:16px;</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Data Inizio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Data Restituzione</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Bid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bidField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Scaffale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rackField">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Palchetto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shelfField">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Posizione</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="positionField">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Indirizzo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>Città</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cityField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="postalCodeField">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent</string>
    </property>
    <property name="text">
     <string>CAP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="movtypeField">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>550</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
